ivy/functional/ivy/gradients.py

```diff
@@ -400,6 +426,63 @@ def optimizer_update(
     -------
     ret
         The new function weights ws_new, following the optimizer updates.
+    
+    Functional Examples
+    -------------------
```

Comment:
we don't show any examples where `inplace` and `stop_gradients` are unspecified. We should include examples where the default values are implicitly used.

Reply:
Yeah I exactly added those examples only where default values are implicitly used.
You can refer docstring examples from line 429 onwards for optimizer_update function.

Reply:
please re-read the comment above and add examples without these keyword arguments provided.